packages/react-scripts/template/src/app/app.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './app';
```

Comment:
In general, we try to adhere to community conventions. However occasionally we have a good reason not to.  I believe this is one of those times.